### ISSUE TITLE
False negative in certain websites

### ISSUE THREAD

**Issue by dennypradipta** [dyaa/ssl-checker#400]
Hi, I am one of the maintainers at [Monika](https://github.com/hyperjumptech/monika), a synthetic monitoring tool. We have been using your library for while, and I have a problem:

So we are trying to check government websites' SSL certificates, but the library said that the certificates are invalid. But, when we opened the website, it shows a valid certificate (according to Chrome).

Here are the two websites that show invalid certs:
- https://www.tangerangselatankota.go.id
- https://www.kemendagri.go.id

I want to know exactly how did you check for the "certificate validity"?

**Comment by dennypradipta** [comment]
For more context, here is how we use your library:

```
import sslChecker from 'ssl-checker';

export async function TLSChecker(url: string) {
  try {
    const domain = new URL(url);
    const { hostname, protocol } = domain;

    if (url.startsWith('http://localhost')) {
      return {
        success: false,
        message: 'Cannot check the TLS of a localhost domain',
        daysRemaining: null,
        valid: false,
        validTo: null
      };
    }

    if (protocol !== 'https:') {
      return {
        success: false,
        message: 'Cannot check a non-HTTPS domain',
        daysRemaining: null,
        valid: false,
        validTo: null
      };
    }

    const hostOptions = { method: 'GET', port: 443 };
    const { daysRemaining, valid, validTo } = await sslChecker(
      hostname,
      hostOptions
    );

    return {
      success: true,
      daysRemaining,
      valid,
      validTo
    };
  } catch (error: any) {
    return {
      success: false,
      message: error.message,
      daysRemaining: null,
      valid: false,
      validTo: null
    };
  }
}
```
